distribution/Dockerfile-ubi-packer

```diff
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN curl --location --output /etc/yum.repos.d/hashicorp.repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+RUN microdnf install python3 rust cargo python3-devel openssl-devel
```

Comment:
Hmm, why do we need rust?

Reply:
To get ansible to install properly :/ Getting ansible in an unsubscribed ubi container is hard, installing it from epel doesn't work because there's missing dependencies.